app/src/main/java/com/del/delcontainer/ui/settings/SettingsFragment.java

```diff
@@ -85,4 +96,12 @@ public void onChanged(@Nullable String s) {
 
         return root;
     }
+
+    /**
+     * Calls the del-api service to get the first name linked to the current user
+     */
+    private void getFirstName(){
+        //settingsViewModel.getUserFirstName(authRepository.getAccessToken(), authRepository.getUserId());
```

Comment:
Get rid of commented out code, please!

Reply:
I kept it for testing but forgot to remove it. I’ll delete those.